cmd/zed/zed.d/zed-functions.sh

```diff
@@ -417,7 +417,7 @@ zed_notify_slack_webhook()
 
     # Construct the JSON message for posting.
     #
```

Comment:
[nit] I'd remove outermost `"` to reduce clutter.

Reply:
re nit: Do you mean this?
```
-    msg_json="$(printf '{"text": "*%s*\\n%s"}' "${subject}" "${msg_body}" )"
+   msg_json=$(printf '{"text": "*%s*\\n%s"}' "${subject}" "${msg_body}" ) 
```
I'm happy to do it, however, it follows the style done else where in that file. Should all of those changes be done in another commit?

Reply:
In the light of your argument I dismiss mine.